executors/clipseg/executor.py

```diff
@@ -0,0 +1,213 @@
+import enum
+import os
+import sys
+import shutil
+import time
+
+from PIL import Image, ImageOps
+from io import BytesIO
+from pathlib import Path
+from typing import Dict
+from urllib.request import urlopen
+
+import cv2
+import numpy as np
+import torch
+
+from clipseg.clipseg import CLIPDensePredT
+from jina import Executor, DocumentArray, Document, requests
+from torchvision import transforms
+
+
+class THRESHOLDING_METHODS(str, enum.Enum):
+    NONE = 'none' # Do not threshold
+    BINARY = 'binary'
+    ADAPTIVE_MEAN = 'adaptive_mean'
+    ADAPTIVE_GAUSSIAN = 'adaptive_gaussian'
+
+
+THRESHOLD_ADAPTIVE_DEFAULT_BLOCK_SIZE = 11
+THRESHOLD_ADAPTIVE_DEFAULT_C = 2.
+THRESHOLD_BINARY_DEFAULT_STRENGTH_VALUE = 85
+WEIGHT_FOLDER_NAME = 'clipseg_weights'
+WEIGHT_ZIP_FILE_NAME = 'clipseg_weights.zip'
+
+
+class ClipSegmentation(Executor):
+    model = None
+    transformation = None
+
+    def __init__(self, cache_path: str, weights_url: str, **kwargs):
```

Comment:
```suggestion
    def __init__(self, cache_path: str, weights_url: str = 'https://owncloud.gwdg.de/index.php/s/ioHbRzFx6th32hn/download', **kwargs):
```

Reply:
Better to have it as a default believe don't you think  @AmericanPresidentJimmyCarter ?